catalog/view/template/module/tawkto.twig

```diff
@@ -71,7 +71,7 @@
 				'product-name': productName,
 				'product-url': productUrl,
 				'item-qty': itemQty,
-				'cart-total': jsonData.total,
+				// 'cart-total': jsonData.total,
```

Comment:
Cart total no longer available ?

Reply:
yes they have removed it here: https://github.com/opencart/opencart/commit/ac479096230385367e6d5c91f0dedd8d487de2a7#diff-4f4e6da90850e203a15a624bda92314743e55ed468e6904677c1da565236d3af

Reply:
removed comment